build/musicbrainz-dev/scripts/createdb.sh

```diff
@@ -91,10 +99,12 @@ if [[ -a /media/dbdump/"${DUMP_FILES[0]}" ]]; then
     mkdir -p $TMP_DIR
     cd /media/dbdump
 
-    # if the import fails because the DB does not exist yet such as when the DB
-    # has been dropped, InitDb will be called again with the create flag
-    /musicbrainz-server/admin/InitDb.pl --echo --import -- --skip-editor --tmp-dir $TMP_DIR ${DUMP_FILES[@]} ||
-    /musicbrainz-server/admin/InitDb.pl --create --echo --import -- --skip-editor --tmp-dir $TMP_DIR ${DUMP_FILES[@]}
+    INITDB_OPTIONS='--echo --import'
+    if ! /musicbrainz-server/script/database_exists MAINTENANCE
+    then
```

Comment:
```suggestion
    if ! /musicbrainz-server/script/database_exists MAINTENANCE; then
```
I usually know this as this but don't know if this is relevant.

Reply:
It is the same thing, I usually prefer line break over semi-colon but since it is already used in the rest of this script, I just followed your suggestion in https://github.com/metabrainz/musicbrainz-docker/pull/125/commits/468aee2c7e615708c3b0a578f70c9f9417f2d59b, thanks!